content/en/bits_ai/getting_started.md

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
One last thing that I forgot earlier: Bits AI is also available on the Datadog mobile app!

Reply:
I added this information to the main page, but if you want to add more detailed instructions to this page for using Bits AI in the mobile app, including a screenshot of the mobile app UI, we can open a new ticket to include that info. Thank you!